nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/ExecuteStreamCommand.java

```diff
@@ -186,7 +195,15 @@ public ValidationResult validate(String subject, String input, ValidationContext
             .addValidator(StandardValidators.createDirectoryExistsValidator(true, true))
             .required(false)
             .build();
-
+    static final PropertyDescriptor REDIRECT_ERROR = new PropertyDescriptor.Builder()
+            .name("Redirect Error Stream")
+            .description("where shoud the error stream from external process get redirected"
+                    + "\n1) log - outputs to the nifi logger"
+                    + "\n2) output stream - redirects to the output stream relation"
+                    + "\n3) error stream - redirects to the error stream relation")
+            .allowableValues("log", "output stream","error stream")
```

Comment:
It would be better to hold these strings in static final variables so they can be reused later without fear of mistyping.

Also spacing looks off here. You should run autoformat again.